src/utils/errors.js

```diff
@@ -0,0 +1,54 @@
+import { app, dialog, shell } from 'electron'
+
+const issueTemplate = (e) => `Please describe what you were doing when this error happened.
+
+**Specifications**
+
+- **OS**: ${process.platform}
+- **IPFS Desktop Version**: ${app.getVersion()}
+- **Electron Version**: ${process.versions.electron}
+- **Chrome Version**: ${process.versions.chrome}
+
+**Error**
+
+\`\`\`
+${e.stack}
+\`\`\`
+`
+
+export function genericError (e) {
```

Comment:
```suggestion
export function showErrorMessage (e) {
```
In general, function names should say what the **do**. more verby than nouny.